packages/experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -14,7 +14,12 @@ const _SelectedItemsList = <TItem extends BaseSelectedItem>(
   }, [props.selectedItems]);
 
   const removeItems = (itemsToRemove: TItem[]): void => {
-    updateItems(items.filter(item => itemsToRemove.indexOf(item) === -1));
+    const updatedItems: TItem[] = [...items];
+    itemsToRemove.forEach(item => {
```

Comment:
Let's make a comment here why we did it this way instead of the way it was originally (so anyone coming in won't change it without knowing the context)